tests/test_builders/xpath_util.py

```diff
@@ -19,24 +20,48 @@ def _gettext(node: Element) -> str:
     return ''.join(n.tail or '' for n in node)
 
 
+def _prettify(nodes: Iterable[Element]) -> str:
+    parts = []
+    for index, node in enumerate(nodes):
+        parts.append(f'(i={index}) ')
+        block = tostring(node, encoding='unicode', method='html')
+        parts.append(textwrap.shorten(block, width=640) + '\n')
+    return ''.join(parts)
```

Comment:
```suggestion
    maxDiff = 640  # NoQA: N816 = https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff
    parts = []
    for index, node in enumerate(nodes):
        parts.append(f'(i={index}) ')
        block = tostring(node, encoding='unicode', method='html')
        parts.append(textwrap.shorten(block, width=maxDiff) + '\n')
    return ''.join(parts)
```

Reply:
After some thoughts, I think I'll even not put a maxwidth (tomorrow). Because we might really want everything to be shown (and it should be a sporadic failure).